src/components/AssessmentCard/AssessmentScoreCard.jsx

```diff
@@ -1,52 +1,16 @@
 import React from 'react';
 import AssessmentRow from './AssessmentRow';
+import './AssessmentScoreCard.css';
+import TextArea from 'react-expanding-textarea';
```

Comment:
🚫 **[eslint]** <[import/order](https://github.com/import-js/eslint-plugin-import/blob/v2.25.2/docs/rules/order.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`react-expanding-textarea` import should occur before import of `./AssessmentRow`